.gitignore

```diff
@@ -1,2 +1,3 @@
 .phpunit.result.cache
 .cow.pat.json
+.vscode
```

Comment:
```suggestion
```

Reply:
We don't tend to have IDE-specific files in the .gitignore for any repo

Reply:
no harm in keeping there though

Reply:
Are we going to start accepting these for any PR that comes through then? I'd like to have a uniform approach to this - either we accept these changes in PRs or we don't.

Reply:
I'd say we should, I've seen these IDE files occasionally committed into repos by accident

Reply:
Okie doke. Will start accepting these from now on 👍